src/features/grading/GradingTypes.ts

```diff
@@ -69,6 +104,69 @@ export type GradingQuery = {
   assessment: GradingAssessment;
 };
 
+export type GradingSubmissionTableProps = {
+  showAllSubmissions: boolean;
+  totalRows: number;
+  pageSize: number;
+  submissions: GradingOverview[];
+  updateEntries: (page: number, filterParams: Object) => void;
+};
+
+export enum ColumnName {
+  assessmentName = 'Name',
+  assessmentType = 'Type',
+  studentName = 'Student(s)',
+  studentUsername = 'Username(s)',
+  groupName = 'Group',
+  progressStatus = 'Progress',
+  xp = 'Raw XP (+Bonus)',
```

Comment:
```suggestion
  xp = 'Raw XP (+ Bonus)',
```
Just a very tiny unimportant personal preference, feel free to reject based on whichever looks better in the UI 😅

Reply:
I think the one without space looks better in the UI after trying it out, having a space feels a little off